### ISSUE TITLE
Direct media function

### ISSUE THREAD

**Issue by simonsavoca** [simonsavoca/partyz#45]
Need an upload page with following properties:

- One click to enable take picture
```
<form action="server.cgi" method="post" enctype="multipart/form-data">
  <input type="file" name="video" accept="video/*" capture="environment">
  <input type="submit" value="Upload">
</form>
```
- Direct upload after taking picture

**Comment by simonsavoca** [comment]
Need to redefine DropZone options block:

```
<script>
            // "myAwesomeDropzone" is the camelized version of the HTML element's ID
            Dropzone.options.myDropzone = {
                parallelUploads: 1,
                paramName: "myFile", // The name that will be used to transfer the file
                acceptedFiles: "image/*",
                maxFilesize: 10,
                capture: "environment",
                dictDefaultMessage: "Appuyez pour séléctionner les photos.",
                init: function() {
                    this.on("queuecomplete", function () {
                        this.setTimeout(this.removeAllFiles(), 5000);
                    });
                }                
            };
        </script>
```

**Comment by simonsavoca** [comment]
Fixed in [v2.2 branch](https://github.com/simonsavoca/partyz/tree/dev_v2.2)
